docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
+
+Knative Services are automatically given a default domain name based on the
+cluster configuration, e.g. "mysvc.mynamespace.mydomain". You can also map a
+single custom domain name that you own to a specific Knative Service using the
+Domain Mapping feature, if enabled.
+
+For example, if you own the "mydomain.com" domain name, and configure its DNS
+to reference your Knative cluster, you can use the DomainMapping feature to
+have this domain be served by a Knative Service.
+
+## Before you begin
+
+1. You need to enable the DomainMapping feature (and a supported Knative
+   Ingress implementation) to use it. See [the Install instructions](../install/).
```

Comment:
where on that page should people look? I didn't see anything covering this topic. Perhaps we could point to the exact page of interest?